.goreleaser.yml

```diff
@@ -79,6 +79,8 @@ nfpms:
     formats:
     - deb
     - rpm
+    scripts:
+      postinstall: "scripts/postinstall.sh"
```

Comment:
Do we still need this with the caveat? Is there anyway we can get this to print out plain text? I'm wondering if we can avoid shipping a command that just prints text

Reply:
we still need something in the form of a script file for APT/RPM installs. Each package manager requires a different way of running something postinstall